apps/src/lab2/views/components/Instructions/ResourcePanel/SettingsPanel.tsx

```diff
@@ -88,21 +96,37 @@ const SettingsPanel: React.FunctionComponent<SettingsPanelProps> = ({
             id={'settings-language-dropdown'}
           />
         </form>
-        {settings.map(setting => (
-          <SimpleDropdown
-            key={setting.id}
-            labelText={setting.label}
-            isLabelVisible={true}
-            onChange={event => setting.onChange(event.target.value)}
-            items={setting.options}
-            selectedValue={setting.selectedValue}
-            name={setting.id}
-            size="s"
-            color={dropdownColor}
-            dropdownTextThickness="thin"
-            className={styles.dropdown}
-          />
-        ))}
+        {settings.map(setting => {
+          const handleChange = (
+            event: React.ChangeEvent<HTMLSelectElement>
+          ) => {
+            sendLab2AnalyticsEvent(
```

Comment:
nit: could we pull `handleChange` outside of the returned code? I find it easier to read methods when they aren't inside the html.